### PR TITLE
Adds draft models for finfings and finding groupings

### PR DESCRIPTION
**Pull request by jamesgaisfordapetito**
Go easy on me. I haven't written code for a long time 🙏

### CODE REVIEW

File: RepoRanger.Models/Finding.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using RepoRanger.Models.Enum;
+
+namespace RepoRanger.Models;
+
+public class Finding
+{
+    public string RepositoryName { get; set; }
+    public Source Source { get; set; }
```
Comment (trichling) [inline RepoRanger.Models/Finding.cs:8]: I would go with just a string here - enums serialization is more difficult (does it go as number or string?) and this is really just a label for us - so extending it for a new source would mean to adjust this package as well which feels unecessary.

File: RepoRanger.Models/FindingGroup.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+using RepoRanger.Models.Enum;
+
+namespace RepoRanger.Models;
+
+public class FindingGroup
+{
+    public IEnumerable<FindingCount> FindingCount { get; set; }
```
Comment (trichling) [inline RepoRanger.Models/FindingGroup.cs:7]: I suggest to add repository name and source here as it makes it easier to access it - if we do not have it we need to go liek

group.Findings.First().Source

File: RepoRanger.Models/Finding.cs (hunk 1 above)
Comment (jamesgaisfordapetito) [reply RepoRanger.Models/Finding.cs]: implemented suggestion 👍

File: RepoRanger.Models/FindingGroup.cs (hunk 2 above)
Comment (jamesgaisfordapetito) [reply RepoRanger.Models/FindingGroup.cs:7]: implemented suggestion 👍

**Comment by trichling** [comment]
looks good to me

**Pull request closed without merging**